music21/romanText/tsvConverter.py

```diff
@@ -599,24 +659,24 @@ def prepStream(self):
             s.insert(0, metadata.Metadata())
 
             firstEntry = self.chordList[0]  # Any entry will do
-            s.metadata.opusNumber = firstEntry.op
-            s.metadata.number = firstEntry.no
-            s.metadata.movementNumber = firstEntry.mov
+            s.metadata.opusNumber = firstEntry.extra.get('op', '')
+            s.metadata.number = firstEntry.extra.get('no', '')
+            s.metadata.movementNumber = firstEntry.extra.get('mov', '')
```

Comment:
rewrite as checks:

```
if 'op' in firstEntry.extra:
    s.metadata.opusNumber = firstExtry.extra['op']
```

The reason is that there's a difference between an opusNumber of '' and no opusNumber defined.  The former will print to musicxml or other places that the opusNumber is ''; the latter will not give an opus number.